docs-layout.hbs

```diff
@@ -30,6 +30,7 @@
       var runnerScript = '/internals/example-runner.js';
       var plunkerBluePrintPath = '/internals/plunker/';
       var npmUrl = '/npm';
+      var trackjs = {{trackjs}};
```

Comment:
This results in a syntax error (`var trackjs = ;`) in projects that run `gulp docs`. You can test this in e.g. kendo-angualar-dialog, after linking the common-tasks repo.